src/main/java/Pow.java

```diff
@@ -0,0 +1,14 @@
+package io.tokenanalyst.udfs;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+
+import java.lang.Math;
+
+@UdfDescription(name = "pow", description = "calculates pow of a value")
+public class Pow {
+    @Udf(description = "pow of a base with given exponent")
+    public double pow(final double base, final double exponent) {
+      return Math.pow(base, exponent);
```

Comment:
this will only work when both the inputs are double, can we also include cases when
1. pow(double, int)
2. pow(int, int)

Reply:
yep, i'll add